docs/state_machines.md

```diff
@@ -0,0 +1,38 @@
+# State Machines
+
+## Statement
+
+```mermaid
+---
+title: State
+---
+stateDiagram-v2
+    [*] --> open
+    open --> payable: mark_payable
+    payable --> paid: mark_paid
```

Comment:
I like this.

I think we can automate it. Looking at the gem we can get a list of all possible states like this:

```ruby
Statement.state_machines[:state].states.map(&:name)
=> [:open, :payable, :paid]
```

Then we can loop through and do something [similar to the official gem](https://github.com/state-machines/state_machines-graphviz/blob/master/lib/state_machines/graphviz/monkeypatch.rb) but build up Mermaid transitions.

Could even be a rake task where we find and replace in this file.